lib/dry/configurable/extension.rb

```diff
@@ -36,8 +34,8 @@ def included(klass)
           prepend(Initializer)
```

Comment:
Now that these lines are reachable, things are more tricky. `include` and `prepend` work differently, `include` will add a module to the ancestors' chain only once, whereas `prepend` does it every time. We should keep this in mind

Reply:
Good call. I'll look to expand the testing to cover a few more possible edge cases before I mark this ready for a final review.